spec/features/home_spec.rb

```diff
@@ -44,19 +44,19 @@
       end
     end
 
-    context '自分がチェックインしているとき' do
+    context '自分が記録しているとき' do
       let!(:checkin) { create(:checkin, user: user, comment: 'おもしろかったよ') }
 
       before do
         visit '/'
       end
 
-      it 'アクティビティに自分のチェックイン情報が表示されること', js: true do
+      it 'アクティビティに自分の記録情報が表示されること', js: true do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.